src/app/components/link-editor-popup/index.tsx

```diff
@@ -126,6 +126,16 @@ export class LinkNodeView implements NodeView {
     this.dom.classList.add('ProseMirror-selectednode');
   }
 
+  removeMark() {
```

Comment:
This I don't think is working as intended, as it seems to be removing the link when I double click. If you follow these steps...

1. Open the editor, and click edit on 'Fred Atherton'
2. On the bio, single click the link text to display the 'edit link' window.
3. Click the link text again, and suddenly the link is lost.